### DISCUSSION TITLE
Export fails with attachments in workspace is larger than maximum export size

### DISCUSSION THREAD

**Discussion by arthar360** [outline/outline#7640] [Self Hosting]
Hi,  On the "Exports" page when I export, I get

> Failed • 4.38 GB of attachments in workspace is larger than maximum export size of 4.10 GB.. Check server logs for more details. • 

In the settings I have
```
AWS_S3_UPLOAD_MAX_SIZE: "262144000"
FILE_STORAGE_UPLOAD_MAX_SIZE: "1024000000"
```

There is nothing special in server logs. There are no restrictions on S3 side.

What could be the problem?

![Screenshot 2024-09-23 at 10 18 40](https://github.com/user-attachments/assets/6045ce31-4a50-4a7d-aca8-9c0f94f8ef8e)

**Comment by tommoor** [comment]
The maximum export size defaults to the total memory available in the system, you can override this with `MAXIMUM_EXPORT_SIZE` environment variable – however if you set it larger than the memory available to the container it will likely crash.
